### PR TITLE
update tag_selection_test weekly graph counter to use 4.weeks instead of 1.year

### PR DESCRIPTION
**Pull request by jywarren**
Attempt to fix https://github.com/publiclab/plots2/issues/9324

### CODE REVIEW

**Comment by jywarren** [comment]
Oops!

```
Progress: |================================ FAIL TagSelectionTest#test_graph (16.57s)
        Expected: 12
          Actual: 14
        test/unit/tag_selection_test.rb:14:in `block in <class:TagSelectionTest>'
```

**Comment by jywarren** [comment]
Maybe the end time does need to be frozen with TimeCop. What if we didn't start with /now/ but went back in time a little? The problem is that most of the nodes will have been created "now" because the tests start up so fast.

**Comment by jywarren** [comment]
Wow lol at least it's consistent...

```
Progress: |======================================= FAIL TagSelectionTest#test_graph (17.90s)
        Expected: 12
          Actual: 14
        test/unit/tag_selection_test.rb:15:in `block in <class:TagSelectionTest>'
```

**Comment by cesswairimu** [comment]
@jywarren I think that may have fixed it

File: test/unit/tag_selection_test.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class TagSelectionTest < ActiveSupport::TestCase
 
   test 'graph' do
-    end_time = Time.now
-    start_time = end_time - 1.year
+    start_time = (Date.today - 1.month).to_time
```
Comment (jywarren) [inline test/unit/tag_selection_test.rb:6]: whoa awesome!

**Comment by jywarren** [comment]
Passed! rerunning again!

**Comment by jywarren** [comment]
OK TY Cess!!!!

**Comment by RuthNjeri** [comment]
Just seeing this now, great job @cesswairimu and @jywarren 🎉 ...I  think the upgrade from .4weeks to .1 month makes a lot of sense...some months have 5 weeks...

**Pull request merged**